README.md

```diff
@@ -86,7 +107,7 @@ ESS Community comes with the following components out-of-the box:
 - [Element Web](https://github.com/element-hq/element-web): The Matrix Web Client provided by Element.
 - PostgreSQL: An optional packaged PostgreSQL server that allows you to quickly set up the stack out-of-the-box. For a better long-term experience, please consider using a dedicated PostgreSQL server. See the [advanced setup docs](./docs/advanced.md) for more information.
 - HAProxy: Provides the routing to Synapse processes.
-- .well-known delegation: Required for federation and Matrix clients.
+- .well-known delegation: Required for federation and Matrix clients
```

Comment:
```suggestion
- `.well-known` delegation: Required for federation and Matrix clients.
```

I'd keep the full stop as all the other bullets, except for Synapse :facepalm:, have them

Reply:
Fine, changed.